package matrices;

import static org.junit.Assert.*;
import static org.junit.Assume.assumeNoException;

import org.junit.Test;

public class Test_task {

	@Test//(expected = Exception.class)
	public void testException() {
		boolean thrown = false;
        Matriz m1 = new Matriz(1, 2, true);
        System.out.println(m1);
        Matriz m2 = new Matriz(3, 4, true);
        System.out.println(m2); 
        try{
        	System.out.println(Matriz.sumarDosMatrices(m1, m2));
        } catch(DimensionesIncompatibles e) {
        	thrown = true;
        }
        assertTrue(thrown);
	}
	
	
}
